client/src/pages/MatchRoomPage.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactElement, FC } from 'react';
+import RootLayout from '../layouts/RootLayout';
+import MatchChat from '../components/matchChat/MatchChat';
+
+const MatchRoomPage: FC = (): ReactElement => {
+  return (
+    <section
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        gap: '2rem',
+      }}
+    >
+      <RootLayout />
```

Comment:
this is a bad way to use the RootLayout component 
what is the value of outlet that returns from RootLayout on this case